src/common/components/beta-notification/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import localforage from 'localforage';
+
+import { BETA_NOTIFICATION_TOGGLE } from '../../reducers/beta-notification';
+import Notification from '../vanilla/notification';
+
+
+const SURVEY_LINK = 'https://docs.google.com/forms/d/e/1FAIpQLSeCAKWHb4w-iNrg-YqyiRVMHULDlZMx9bXyK9a7s40sXYjQzQ/viewform?usp=sf_link';
+
+export const BETA_NOTIFICATION_DISMISSED_KEY = 'beta_notification_dismissed';
+
+export class BetaNotificationView extends Component {
+
+  async storeNotificationDismissed() {
+    await localforage.setItem(BETA_NOTIFICATION_DISMISSED_KEY, true);
+  }
+
+  onRemoveClick(event) {
+    event.preventDefault();
+
+    this.props.dismissNotification();
+    this.storeNotificationDismissed();
+  }
+
+  render() {
+    return this.props.isVisible
+      ? (
+        <Notification onRemoveClick={this.onRemoveClick.bind(this)}>
```

Comment:
Simply renders our Vanilla notification.

Component is added to top of the page container - in document flow - meaning it will push down the content when it appears.